### PR TITLE
Fix for Issue #4040 - wording request

### PR DESCRIPTION
**Pull request by rodeten**
## Summary

Describe your changes here.

Fixes dotnet/docs#4040 as suggested.

### CODE REVIEW

**Review by carlossanlop — COMMENTED** [review]
Thank you for submitting the change, @rodeten . As I mentioned in the issue, I think the wording could be improved. As it is right now, the message is not clear to me. Any chance you could reword it? @gewarren ideas are welcome.

File: xml/System/String.xml [hunk 1]
```diff
@@ -12583,7 +12583,7 @@ The following example defines a `StripStartTags` method that uses the <xref:Syst
 
 -   If the character or character sequence is not included in the end of the substring, the `length` parameter equals `endIndex` - `startIndex`, where `endIndex` is the return value of the <xref:System.String.IndexOf%2A> or <xref:System.String.IndexOf%2A> method.
 
- If `startIndex` is equal to zero and  equals the length of the current string, the method returns the original string unchanged.
+ If `startIndex` is equal to zero and results equals the length of the current string, the method returns the original string unchanged.
```
Comment (gewarren) [inline xml/System/String.xml:12586]:
```suggestion
 If `startIndex` is equal to zero and `length` equals the length of the current string, the method returns the original string unchanged.
```

**Comment by mairaw** [comment]
Closing since this is targeting live branch

**Comment by carlossanlop** [comment]
Good catch, @mairaw. Is there a way to forbid pull requests to the live branch except for people with permission? cc @gewarren @BillWagner

**Comment by mairaw** [comment]
Not that I'm aware @carlossanlop.

**Pull request closed without merging**